platform/android/java/lib/patches/com.google.android.vending.licensing.patch

```diff
@@ -32,11 +32,11 @@ import org.godotengine.godot.BuildConfig;
      }
  
 -    assert (e == outBuff.length);
-+    // -- GODOT start --
++    // -- BLAZIUM start --
 +    //assert (e == outBuff.length);
```

Comment:
The lines 35-38 maybe from a different branch, please validate this is required for rebranding